<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1041</width>
    <height>561</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EBAZ4205 SDR Spectrum Analyzer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout" columnstretch="6,0,0">
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <item row="1" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="handleWidth">
       <number>1</number>
      </property>
      <widget class="SpChartView" name="spChartView"/>
      <widget class="QGraphicsView" name="graphicsView_2">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLabel" name="lblFreq">
      <property name="text">
       <string>100.0Mhz</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" colspan="2">
     <widget class="QToolBox" name="toolBox">
      <property name="minimumSize">
       <size>
        <width>280</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>320</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>280</width>
         <height>458</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Tuner</string>
       </attribute>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>261</width>
          <height>171</height>
         </rect>
        </property>
        <layout class="QFormLayout" name="formLayout_3">
         <item row="1" column="0">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>IF Gain</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="comboBoxIFGain">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <item>
            <property name="text">
             <string>1</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>4</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>8</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>16</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>32</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>64</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>128</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>256</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>512</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>1024</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_4">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>NFFT</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Averaging</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QDoubleSpinBox" name="doubleSpinBoxAlfa">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="decimals">
            <number>2</number>
           </property>
           <property name="maximum">
            <double>1.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.050000000000000</double>
           </property>
           <property name="stepType">
            <enum>QAbstractSpinBox::DefaultStepType</enum>
           </property>
           <property name="value">
            <double>0.700000000000000</double>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>IF BW (KHz)</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QComboBox" name="comboBoxIFBandwidth">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="currentText">
            <string>2000</string>
           </property>
           <property name="placeholderText">
            <string/>
           </property>
           <item>
            <property name="text">
             <string>2000</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>1000</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>500</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>250</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>125</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>60</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>30</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>16</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>8</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>4</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>1</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_6">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>FFT Window</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QComboBox" name="comboBoxWindow">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="editable">
            <bool>false</bool>
           </property>
           <property name="currentIndex">
            <number>4</number>
           </property>
           <item>
            <property name="text">
             <string>rectwin</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>hann</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>hann_per</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>hann_sym</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>hamming</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>hamm_per</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>hamm_sym</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>blackman</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>blac_per</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>black_sym</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>nuttall</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>nutt_per</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>nutt_sym</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>flattop</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>flat_per</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>flat_sym</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>triang</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>bartlett</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>barthann</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>bohman</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>parzen</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>gauss</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>gauss_2p5</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>gauss_3p2</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>tukey</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>tukey_0p0</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>tukey_0p2</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>tukey_0p5</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>tukey_0p8</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>tukey_1p0</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>taylor</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>tayl_4</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>tayl_4_m20</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>tayl4_m40</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>tayl_5</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>tayl_5_m20</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>tayl_5_m30</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>tayl_5_m40</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>tayl_6</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>tayl_6_m20</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>tayl_6_m30</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>tayl_6_m40</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>kaiser</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>kais_0p5</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>kais_0p8</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="formLayoutWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>240</y>
          <width>271</width>
          <height>127</height>
         </rect>
        </property>
        <layout class="QFormLayout" name="formLayout_2">
         <item row="1" column="0">
          <widget class="QLabel" name="label_8">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Tuned Freq (Hz)</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="spinBoxTunedFrequency">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="minimum">
            <number>100000</number>
           </property>
           <property name="maximum">
            <number>32000000</number>
           </property>
           <property name="displayIntegerBase">
            <number>10</number>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QSpinBox" name="spinBoxTestFrequency">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="minimum">
            <number>1000000</number>
           </property>
           <property name="maximum">
            <number>32000000</number>
           </property>
           <property name="singleStep">
            <number>1000000</number>
           </property>
           <property name="displayIntegerBase">
            <number>10</number>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Test Gen (Hz)</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QRadioButton" name="radioButtonADC">
           <property name="text">
            <string>ADC in</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QRadioButton" name="radioButtonGenerator">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Test Gen In</string>
           </property>
           <property name="iconSize">
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QCheckBox" name="cbAreaSeries">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>0</y>
          <width>171</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Area Series</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="cbCaptureRF">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>161</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Capture RF</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="toolBoxLevel">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>280</width>
         <height>458</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Waterfall</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SpChartView</class>
   <extends>QGraphicsView</extends>
   <header>spchartview.h</header>
   <slots>
    <slot>change_fft_window(QString)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
